optimum/intel/openvino/modeling.py

```diff
@@ -575,3 +589,234 @@ def forward(
         outputs = self.request(inputs)
         logits = torch.from_numpy(outputs["logits"]).to(self.device) if not np_inputs else outputs["logits"]
         return SequenceClassifierOutput(logits=logits)
+
+
+CTC_EXAMPLE = r"""
+    Example of CTC:
+
+    ```python
+    >>> from transformers import {processor_class}
+    >>> from optimum.intel.openvino import {model_class}
```

Comment:
Awesome work @helena-intel ! Could you add all these classes in [here](https://github.com/huggingface/optimum-intel/blob/f2fdce2fd4c85a2d0ce53d3f4c653aa49120fdbe/optimum/intel/openvino/__init__.py#L37), [here](https://github.com/huggingface/optimum-intel/blob/f2fdce2fd4c85a2d0ce53d3f4c653aa49120fdbe/optimum/intel/__init__.py#L89) and create dummy classes [here](https://github.com/huggingface/optimum-intel/blob/f2fdce2fd4c85a2d0ce53d3f4c653aa49120fdbe/optimum/intel/utils/dummy_openvino_objects.py#L18) so that we can import them from `optimum.intel` directly :
```suggestion
    >>> from optimum.intelimport {model_class}
```

Reply:
Thank you, done!